api/controllers/userController.ts

```diff
@@ -0,0 +1,13 @@
+import { Request, Response } from "express";
+import userService from "../services/userService";
+import { asyncErrorHandler } from "../middlewares/error";
+
+class user {
```

Comment:
```suggestion
class userController {
```
로 변경해주시는게 어떨까 싶습니다 ㅎ